content/blog/2023/nextflow-with-gbatch.md

```diff
@@ -0,0 +1,291 @@
+    fastqc_ch = FASTQC(read_pairs_ch)
+    MULTIQC(quant_ch.mix(fastqc_ch).collect())
+}
+
+workflow.onComplete {
+    log.info ( workflow.success ? "\nDone! Open the following report in your browser --> $params.outdir/multiqc_report.html\n" : "Oops .. something went wrong" )
+}
+```
+
+> More information about this pipeline and where this came from can be found in the public training material developed by Seqera Labs at [https://training.seqera.io/](https://training.seqera.io/).
+
+#### Download the input data
+
+Create a folder named data in your project folder and download the example files (also taken from the Seqera training material).
+
+```bash
+$ mkdir data; cd data
+$ wget https://raw.githubusercontent.com/seqeralabs/nf-training-public/master/nf-training/data/ggal/gut_1.fq
+$ wget https://raw.githubusercontent.com/seqeralabs/nf-training-public/master/nf-training/data/ggal/gut_2.fq
+$ wget https://raw.githubusercontent.com/seqeralabs/nf-training-public/master/nf-training/data/ggal/transcriptome.fa
+$ cd ..
+```
+
+Files can also be uploaded directly to Google Cloud Storage, or downloaded directly from GitHub, depending on your own preferences.
+
+Use the command below to authenticate the Google Cloud Platform and direct Nextflow to use your account.
+
+```bash
+$ gcloud auth application-default login
+```
+
+#### Fire! Run the pipeline!
+
+With that done, you’re now ready to run the example Nextflow pipeline. Remember the name of the bucket you created? You will now use it to tell Nextflow here to move our input data and store intermediate and final data. That is, we're setting our work directory with the `-w` parameter.
+
+```bash
+$ nextflow run main.nf -w gs://rnaseq-pipeline-bckt/scratch
+```
+
+
+It's good practice not to write to the root directory of your bucket. It is better to write to another folder, such as the scratch folder within it. Don't worry about everything being executed remotely in the bucket because the `publishDir` directive was used in the `main.nf` script to direct results into a local folder called `results`.
+
+In your command line, you should see something like this:
+
+![Nextflow ongoing run on Google Cloud Batch](/img/ongoing-nxf-gbatch.png)
+
+
+You can open another terminal and check the status of your jobs on Google Batch with the following command:
+
+```bash
+$ gcloud batch jobs list
+```
+
+By the end of it, if everything worked well, you should see something like:
+
+![Nextflow run on Google Cloud Batch finished](/img/nxf-gbatch-finished.png)
+
+And that's it, folks! 😆
+
+You will find more information about Nextflow on Google Batch in [this blog post](https://www.nextflow.io/blog/2022/deploy-nextflow-pipelines-with-google-cloud-batch.html), and in [the official documentation of Nextflow](https://www.nextflow.io/docs/latest/google.html). Don't forget to check [the Google Cloud official docs on running Nextflow pipelines with it](https://cloud.google.com/life-sciences/docs/tutorials/nextflow#gcloud)!
```

Comment:
```suggestion
You will find more information about Nextflow on Google Batch in [this blog post](https://www.nextflow.io/blog/2022/deploy-nextflow-pipelines-with-google-cloud-batch.html) and the [official Nextflow documentation](https://www.nextflow.io/docs/latest/google.html).
```

Reply:
I don't think we should link to the old Life Sciences tutorial because it might be confusing. The Google bits should still be correct but the Nextflow bits won't be.

Reply:
So you think we shouldn't mention it at the end, or don't mention it at the beginning either?

Reply:
I don't see the Life Sciences tutorial at the beginning, but yeah I don't think we should mention it at all

Reply:
Oh, I see now!